code/part-one/blockchain.js

```diff
@@ -41,14 +41,14 @@ class Block {
       - the previous hash
       - the block timestamp
   */
- calculateHash() {
-  return SHA512(`
-    ${this.previousHash}
-    ${this.timestamp}
-    ${JSON.stringify(this.transactions)}
-    ${this._nonce}
-  `).toString();
-}
+  calculateHash() {
+    return SHA512(`
+      ${this.previousHash}
+      ${this.timestamp}
+      ${JSON.stringify(this.transactions)}
+      ${this._nonce}
+    `).toString();
```

Comment:
This use of a template string seems problematic to me. Since the newlines and indents will be included in the string, if someone were to refactor the use of whitespace later, it would break the blockchain. I would prefer concatenation here.